packages/base/src/commands.ts

```diff
@@ -1087,25 +1057,32 @@ export function addCommands(
   });
 
   commands.addCommand(CommandIDs.updateCameraSettings, {
-    label: trans.__('Camera Settings'),
+    label: () => {
+      const isOn =
+        tracker.currentWidget?.content?.cameraSettings?.type === 'Perspective';
+      return isOn
+        ? trans.__('Perspective projection is on')
+        : trans.__('Orthographic projection is on');
+    },
     isEnabled: () => Boolean(tracker.currentWidget),
-    icon: cameraSolidIcon,
+    icon: videoSolidIcon,
+    isToggled: () => {
+      const current = tracker.currentWidget?.content;
+      return current?.cameraSettings.type === 'Orthographic';
+    },
     execute: async () => {
       const current = tracker.currentWidget;
-
       if (!current) {
         return;
+      } else {
+        const currentSettings: CameraSettings = current.content.cameraSettings;
+        if (currentSettings.type === 'Perspective') {
+          current.content.cameraSettings = { type: 'Orthographic' };
+        } else {
+          current.content.cameraSettings = { type: 'Perspective' };
+        }
```

Comment:
Last tiny nitpick, in case we add more camera settings later, we don't want to overwrite the full settings here

```suggestion
        if (currentSettings.type === 'Perspective') {
          current.content.cameraSettings.type = 'Orthographic';
        } else {
          current.content.cameraSettings.type = 'Perspective';
        }
```